pkg/policy/api/l4.go

```diff
@@ -63,6 +102,26 @@ type PortRule struct {
 	// +optional
 	Ports []PortProtocol `json:"ports,omitempty"`
 
+	// TerminatingTLS is the TLS context for the connection terminated by
+	// the L7 proxy.  For egress policy this specifies the server-side TLS
+	// parameters to be applied on the connections originated from the local
+	// POD and terminated by the L7 proxy. For ingress policy this specifies
```

Comment:
nit: This API wording usually uses the Cilium `endpoint` terminology rather than `POD`. (Same below)

Reply:
Changed, thanks!